### ISSUE TITLE
React Web App?

### ISSUE THREAD

**Issue by r-saba** [ArchambaultP/BillProject#2]
Will this be a react web app?
Have some views that need to be done (Login view, camera view, saved bills view ... etc) ?

**Comment by notarock** [comment]
It is. @ArchambaultP ? :ping_pong:

**Comment by ArchambaultP** [comment]
Yes, the plan is to build the frontend in react, and the backend using flask.
